### ISSUE TITLE
Using no type assertion with var is not reported

### ISSUE THREAD

**Issue by ozon2** [gostaticanalysis/fourcetypeassert#8]
This is a simple example with no type assertion that `forcetypeassert` fails to find:
```go
var a interface{}
var _ int = a.(int)
```
